src/templates/base/store/global/reducer.js

```diff
@@ -0,0 +1,19 @@
+import * as actions from './actions';
+
+// @Initial State
+export const initialState = {
+    loading: true,
```

Comment:
Please write code to at least update them

Reply:
done.